About/FAQ.md

```diff
@@ -190,4 +206,118 @@ TO BE WRITTEN
 What happens if I lose/break my security key
 ----
 
-TO BE WRITTEN
+This depends on whether or not you have backups for the key.  If you have no backup you will have lost access to all data that was encrypted using this key.  If you have backups you may create a new key from them.  The counter on the key will be different but you can reset that in heads.  Since you know that something changed you can ignore the tampering warning.
+
+What is the recovery shell?
+----
+
+The recovery shell is a minimal unix shell running on top of the heads kernel.  It can be used to investigate and configure the heads environment.
+
+What are the limitations of the recovery shell? TBD
+----
+
+This thread (https://github.com/osresearch/heads/issues/639#issuecomment-570014587) says that the secrets from nvram are wiped before the recovery shell is launched.  Does this limit what the recovery shell can do?  Can the secrets be recovered using my other security infrastructure for validating that each step of the configuration and boot is working?
```

Comment:
Going into recovery shell wipes the secret into ram and extends the PCRs so the calculations from the PCRs are invalidated: the result of the measurements are not matching the state required to have a matching OTP/HOTP at that point.

Code explains it better, as pointed in that ticket you are referring. That part should probably continue to be discussed there. Tag me there.